src/main/java/BinaryTree/BinarySearchTreeTest.java

```diff
@@ -0,0 +1,61 @@
+package BinaryTree;
+
+/**
+ * Created by jaewonlee on 2020-02-29.
+ */
+public class BinarySearchTreeTest {
+    public static void main(String[] args) {
+
+        /*
+        *   BinarySearchTree(BST)
+        *
+        *   - BST is a binary trees that keeps their keys in sorted order.
+        *   - SearchKey method returns true if a node in tree contains the target key value
+        *   - DeleteNode removes target node from the tree
+        *   - InsertNode inserts a new node into the tree depending on the new node's key
+        *   - (left : key < root, right : key > root)
+        *
+        *   Minimum height : O(logN)
+        *   Maximum height : O(N)
+        *   Minimum search time : O(log2(N))
+        *   Maximum Search time : O(N)
+        *   Minimum Insertion time : O(log2(N))
+        *   Maximum Insertion time : O(N)
+        *
+        *   Below example tree has a height of 4
+        *                   5
+        *                 /   \
+        *               2       7
+        *              / \     / \
+        *             1   3   6   9
+        *                  \
+        *                   4
+        *
+        *   program output:
+        *   52713694
+        *   Is 4 in the tree? : true
+        *   Remove 9 from the tree
+        *   5271364
+        */
+
+        BinarySearchTree bsTree = new BinarySearchTree(5);
+        // Insert a node
+        bsTree.insertNode(new BinaryNode(2));
```

Comment:
bsTree.inseart(2)
bsTree.inseart(1)
...

Must not be creating a Node instance outside of Tree class